composer.json

```diff
@@ -96,7 +96,8 @@
         },
         "files": [
             "src/functions/node_helper.php",
-            "src/constants.php"
+            "src/constants.php",
+            "vendor/phpstan/phpstan-extracted/vendor/autoload.php"
```

Comment:
This will load phpstan's autoload and its own `vendor/nikic/php-parser`.
This package is in the same version, so Rector's `preload.php` would conflict with same classes.

That's why its nulled bellow.

Reply:
This line must be added via config CLI command: `composer config ...`

In: `.github/workflows/build_scoped_rector_php70.yaml`

Now it breaks other builds